### PR TITLE
PIM-7125: Document "include_children" option in a remove action

### PR DESCRIPTION
**Pull request by mmetayer**
see https://github.com/akeneo/pim-enterprise-dev/pull/3571

### CODE REVIEW

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 1]
```diff
@@ -282,13 +282,14 @@ Two parameters are required, two other are optional.
...
-- field: attribute code.
-- locale: local code for which value is assigned (optional).
-- scope: channel code for which value is assigned (optional).
-- items: attribute values to remove.
+Two parameters are required, three other are optional.
+ - field: attribute code or "categories".
+ - locale: locale code for which value is assigned (optional).
+ - scope: channel code for which value is assigned (optional).
+ - items: attribute values to remove.
+ - include_children: apply the removal to the subtree of the given categories. Only appliable if ``field`` is set to "**categories**" (optional, defaults to ``false``)
```
Comment (Delphineray) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst]: If yes then apply the removal to the children of the given categories...

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 2]
```diff
@@ -302,6 +303,29 @@ Two parameters are required, two other are optional.
...
+
+        actions:
+            - type: remove
+              field: categories
+              items:
+                - clothing
+              include_children: true
+
+    .. warning::
+
```
Comment (Delphineray) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst]: Could you add also an example to remove from a whole tree (with the root category)?

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 3]
```diff
@@ -282,13 +282,14 @@ Two parameters are required, two other are optional.
...
 
-This action removes values to a multiselect, a category or a collection.
+This action removes values from a multiselect, a category or a collection.
 
-Two parameters are required, two other are optional.
-- field: attribute code.
-- locale: local code for which value is assigned (optional).
-- scope: channel code for which value is assigned (optional).
-- items: attribute values to remove.
+Two parameters are required, three other are optional.
```
Comment (SamirBoulil) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst]: three others ?

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 4]
```diff
@@ -282,13 +282,14 @@ Two parameters are required, two other are optional.
...
+ - field: attribute code or "categories".
+ - locale: locale code for which value is assigned (optional).
+ - scope: channel code for which value is assigned (optional).
+ - items: attribute values to remove.
```
Comment (SamirBoulil) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst]: attribute values ? You mean attribute options ?

File: manipulate_pim_data/rule/general_information_on_rule_format.rst [hunk 5]
```diff
@@ -282,13 +282,14 @@ Two parameters are required, two other are optional.
...
+ - include_children: if ``true``, then also apply the removal to the children of the given categories. Only appliable if ``field`` is set to "**categories**" (optional, defaults to ``false``).
```
Comment (SamirBoulil) [inline manipulate_pim_data/rule/general_information_on_rule_format.rst]: only applicable

Comment (mmetayer) [reply manipulate_pim_data/rule/general_information_on_rule_format.rst]: :white_check_mark:

File: manipulate_pim_data/rule/general_information_on_rule_format.rst (hunk 3 above)
Comment (mmetayer) [reply manipulate_pim_data/rule/general_information_on_rule_format.rst]: :white_check_mark:

File: manipulate_pim_data/rule/general_information_on_rule_format.rst (hunk 4 above)
Comment (mmetayer) [reply manipulate_pim_data/rule/general_information_on_rule_format.rst]: I went with plain 'values to remove' instead

**Pull request merged**